src/gatewayBFF.ts

```diff
@@ -163,18 +163,23 @@ export class GatewayBFF {
         if (fragment) {
             const version = this.detectVersion(fragment, cookie);
             const fragmentContent = await fragment.render(req, version);
-
+            
             const gatewayContent = {
                 content: fragmentContent,
                 $status: +(fragmentContent.$status || HTTP_STATUS_CODE.OK),
                 $headers: fragmentContent.$headers || {},
+                $httpCookies: fragmentContent.$httpCookies || {},
```

Comment:
`$cookies` better I guess.

Reply:
to increase the awareness of what we do, I choosed this naming convention(http cookie vs browser cookie). In this feature, we set to 'Set-Cookie' value of header in http response. But if you are say these are same thing and not confused, we can use $cookies to sake of the simplicity.